app/models/early_career_payments/eligibility_matrix_calculator.rb

```diff
@@ -1,12 +1,5 @@
 module EarlyCareerPayments
   class EligibilityMatrixCalculator
```

Comment:
Didn't mention on previous PR but I think in future we could do better on consolidating and naming eligibility related code.

Reply:
Yes I agree, eligibility is everywhere and its been a challenge as to what to call some methods/classes. I went with the MatrixCalculator as it calcuates if a ECP claims eligibility is actually one of 3 statuses after all of the eligibility questions have been successfully answered. These are:

- Eligible Now (in this claim window/AcademicYear)
- Eligible Later (say in 2022 if in 2021 and meets all criteria)
- No longer eligible